### PR TITLE
strategic-intiatives: adding `npm integration` and `open web standards`

### PR DESCRIPTION
**Pull request by MylesBorins**
adding new initiatives that I would like to take on

* integration of npm in node
* engagement in open web standards

### CODE REVIEW

**Comment by ljharb** [comment]
What does “integration into node” mean?

File: Strategic-Initiatives.md [hunk 1]
```diff
@@ -11,19 +11,20 @@ and have the support needed.
...
+| Workers            | [Anna Henningson][addaleax]                               | https://github.com/nodejs/worker                                 |
+| Error Messages     | [Joyee Cheung][joyeecheung]                               | [node#11273][], [node#18106][]                                   |
+| Core Promise APIs  | [James Snell][jasnell]                                    |                                                                  |
+| TSC Governance     | [Myles Borins][MylesBorins]                               | https://github.com/nodejs/TSC/issues/383                         |
+| New Streams APIs   | [James Snell][jasnell] + [Jeremiah Senkpiel][fishrock123] | https://github.com/nodejs/node/pull/16414                        |
+| V8 Currency        | [Michaël Zasso][targos]                                   |                                                                  |
+| npm Integration    | [Myles Borins][MylesBorins]                               |                                                                  |
```
Comment (richardlau) [inline Strategic-Initiatives.md]: Assuming this is https://github.com/nodejs/TSC/issues/528, perhaps `npm Currency` (as per the line above), or is something different being referred to here?

Comment (MylesBorins) [reply Strategic-Initiatives.md]: TBH I was going to send a PR to update the V8 initiative name. I think integration makes more sense.

**Comment by MylesBorins** [comment]
@ljharb integration as in integrating npm into the node.js source tree

File: Strategic-Initiatives.md (hunk 1 above)
Comment (richardlau) [reply Strategic-Initiatives.md]: Either name works for me, as long as they are consistent if referring to the same goals.

**Comment by ljharb** [comment]
@MylesBorins like copypasta, or something like submodules?

File: Strategic-Initiatives.md [hunk 2]
```diff
@@ -11,19 +11,20 @@ and have the support needed.
...
+| Open Web Standards | [Myles Borins][MylesBorins]                               |                                                                  |
```
Comment (mhdawson) [inline Strategic-Initiatives.md]: It is possible to have at least an issue or something with more details for each one and add a link?

**Comment by mhdawson** [comment]
Going to land as is, @MylesBorins would be good if you could add links in a follow on PR.

**Comment by mhdawson** [comment]
@MylesBorins needs a rebase.

**Comment by mhdawson** [comment]
@MylesBorins could you rebase?

**Comment by mhdawson** [comment]
merged and landed as 88b45a12300bfc38ce02c5bd957d086fe57375ff

**Pull request closed without merging**